File: core/src/org/rexcellentgames/burningknight/debug/DieCommand.java
package org.rexcellentgames.burningknight.debug;

import org.rexcellentgames.burningknight.entity.creature.player.Player;

public class DieCommand extends ConsoleCommand {
	{
		name = "/kill";
		shortName = "/k";
	}

	@Override
	public void run(Console console, String[] args) {
		if (Player.instance != null) {
			Player.instance.die();
		}
	}
}

File: core/src/org/rexcellentgames/burningknight/debug/DieCommand.kt
package org.rexcellentgames.burningknight.debug

import org.rexcellentgames.burningknight.entity.creature.player.Player

class DieCommand : ConsoleCommand("/die", "de", "Death") {
    override fun run(console: Console, args: Array<String>) {
        if (Player.instance != null) {
            Player.instance.die()
        }
    }
}